llvm/lib/MC/MCCASObjectV1.cpp

```diff
@@ -1024,7 +1061,16 @@ Error MCCASBuilder::buildFragments() {
 
     // Handle Debug Info sections separately.
     if (DebugInfoSection == &Sec) {
-      if (auto E = splitDebugInfoCUs())
+      if (auto E = splitDebugInfoCUs(DebugInfoSection))
+        return E;
+      continue;
+    }
+    // If the Section is the DWARF Line Section, append all the contents of the
+    // section into a buffer and split it up into multiple CAS blocks, where one
+    // block represents one function's contribution into the section or one line
+    // table.
```

Comment:
I believe all of these comments would be better as the documentation of what `createLineSection` does, not as inline comments. Note how the current documentation of `createLineSection` is much simpler than the description above.

Reply:
I see, so you are saying I should move this comment to the header file? That works too!

Reply:
Yup, I think it would be easier to find the information that way